### ISSUE TITLE
Drone ID support possible?

### ISSUE THREAD

**Issue by MAVProxyUser** [Speexvocon/FlipperZeroESPexpansion#1]
Is is possible to support this work? 
https://github.com/DJISDKUser/ESP8266_DJI_DroneID_Throwie

**Comment by Speexvocon** [comment]
Yes should be possible! I might be able to add it to the flasher

**Comment by skynet01** [comment]
+1 for this! :) FYI there is a fork of it with some bugs fixes here: https://github.com/EosBandi/ESP8266_DJI_DroneID_Throwie
